nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/resolver/TestDocumentModelResolver.java

```diff
@@ -537,6 +541,28 @@ public void testSerialization() throws Exception {
         assertEquals(idRef, readResolver.getReference(doc));
     }
 
+    @Test
+    public void testUserWithNoAccessToDocument() {
+        String username = "foo";
+        DocumentModelResolver resolver = new DocumentModelResolver();
+        Map<String, String> params = new LinkedHashMap<>();
+        params.put(PARAM_STORE, STORE_ID_REF);
```

Comment:
Use just `HashMap`, not `LinkedHashMap`. And actually here use just `Collections.singletonMap`.